### DISCUSSION TITLE
TVHeadend

### DISCUSSION THREAD

**Discussion by koepcke** [MichaIng/DietPi#5958] [Software Request]
Hi,
is it possible to get the latest release of TVHeadend server.
The new AVM Fritz.Box Firmware > 7.29 needs a newer release to work with.
(> 3b1d7a9 ("SAT>IP client: UPnP header field names are case insensitive") 15.03.22)

**Comment by MichaIng** [comment]
TVHeadend is provided by the Raspberry Pi APT repository, so this is outside of our abilities. Last update (on Bullseye) was to v4.3: https://archive.raspberrypi.org/debian/pool/main/t/tvheadend/

v4.3 is the latest release already, although the version suffix on the official APT repository is a little newer (while the actual package link does not exist 🤔): https://apt.tvheadend.org/unstable/dists/bullseye/main/binary-amd64/Packages (https://apt.tvheadend.org/artifacts/ `5c` sub directory does not exist...)

Ah, I see now the commit, dated after latest package release on RPi repo: https://github.com/tvheadend/tvheadend/tree/3b1d7a9

Looks like you need to compile it yourself then. E.g.:
```sh
cd /tmp
curl -fLO 'https://github.com/tvheadend/tvheadend/archive/refs/heads/master.tar.gz'
tar xf master.tar.gz
rm master.tar.gz
cd tvheadend-master
```
Then according to instructions here: https://tvheadend.org/projects/tvheadend/wiki/Packaging
```sh
./Autobuild.sh -t bullseye-aarch64
apt install ../tvheadend*.deb
```
If it is an RPi 64-bit system.
